src/index.js

```diff
@@ -0,0 +1,53 @@
+import './style.css';
+
+const array = [
+  {
+    description: 'wake up',
+    completed: true,
+    index: 1,
+  },
+
+  {
+    description: 'dress my bed',
+    completed: true,
+    index: 4,
+  },
+
+  {
+    description: 'brush my teeth',
+    completed: true,
+    index: 2,
+  },
+
+  {
+    description: 'clean the house',
+    completed: true,
+    index: 3,
+  },
+  {
+    description: 'bath',
+    completed: true,
+    index: 2,
+  },
+
+  {
+    description: 'get ready for meeting',
+    completed: true,
+    index: 3,
+  },
+
+];
+
+const activities = document.getElementById('do-list');
+for (let i = 0; i < array.length; i += 1) {
+  activities.innerHTML += `
+  
+  <div class="each-row">
+    <div>
+    <input  type="checkbox" class="check" id="input${array[i].index}">
+    <input id="${array[i].index}" class="task-item" value="${array[i].description}"></div>
+    <button id="row-btn"><i class="fa-solid fa-ellipsis-vertical"></i></button>
+  
+</div>
+      `;
+}
```

Comment:
- [ ] According to this project, you're expected to render the list of todos in order of their indexes. It would be nice if you could look and filter the todos to render in order 1.....9. Currently, they're not being rendered in order. I recommend you used the sort() method on the array, and then chain the forEach() method to that so they render in order. 👍🏽

**See The Requirement Below**

<img width="1042" alt="Screenshot 2023-01-18 at 12 24 38 PM" src="https://user-images.githubusercontent.com/50941074/213171651-6ebfcd5a-34d2-45ad-974c-56858f25a301.png">

Reply:
I used 
array.sort((a, b) => {
    return a.index - b.index;
});
to solve it.